tests/test_errenskitchen.py

```diff
@@ -0,0 +1,77 @@
+# mypy: allow-untyped-defs
+
+from recipe_scrapers.errenskitchen import ErrensKitchen
+from tests import ScraperTest
+
+
+class TestErrensKitchenScraper(ScraperTest):
+    scraper_class = ErrensKitchen
+
+    def test_host(self):
+        self.assertEqual("errenskitchen.com", self.harvester_class.host())
+
+    def test_author(self):
+        self.assertEqual("Erren Hart", self.harvester_class.author())
+
+    def test_category(self):
+        self.assertEqual("Dinner", self.harvester_class.category())
+
+    def test_cuisine(self):
+        self.assertEqual("Italian", self.harvester_class.cuisine())
+
+    def test_description(self):
+        self.assertEqual(
+            "A delectable dish of Creamy Sun-Dried Tomato Pasta with Chicken flavored with Parmesan cheese, sun-dried tomatoes, onions, and garlic.",
+            self.harvester_class.description(),
+        )
+
+    def test_image(self):
+        self.assertEqual(
+            "https://www.errenskitchen.com/wp-content/uploads/2022/05/Chicken-Sundried-Tomato-Pasta-06.jpg",
+            self.harvester_class.image(),
+        )
+
+    def test_ratings(self):
+        self.assertEqual(5.0, self.harvester_class.ratings())
+
+    def test_title(self):
+        self.assertEqual(
+            "Creamy Sun-Dried Tomato Pasta with Chicken", self.harvester_class.title()
+        )
+
+    def test_total_time(self):
+        self.assertEqual(30, self.harvester_class.total_time())
+
+    def test_yields(self):
+        self.assertEqual("4 servings", self.harvester_class.yields())
+
+    def test_ingredients(self):
+        expected_ingredients = [
+            "1 tablespoon olive oil",
+            "2 lbs boneless chicken thighs (or breasts, cubed)",
+            "1 onion (chopped)",
+            "¼ cup sun-dried tomatoes (drained and chopped)",
+            "2 tablespoons sun-dried tomato paste",
+            "3 garlic cloves (minced)",
+            "¾ cup chicken stock",
+            "¾ cup freshly grated Parmesan cheese",
+            "1½ cups half and half (or whipping cream)",
+            "1 pound pasta",
+            "1 tablespoon fresh basil (chopped)",
+            "Salt and pepper to taste",
+        ]
+        self.assertEqual(expected_ingredients, self.harvester_class.ingredients())
+
+    def test_instructions(self):
+        expected_instructions = (
```

Comment:
Suggestion: maybe turn this into a `list` constant, so that you can omit the `\n` at the end of each line?

Reply:
I'm a little lost at the implementation of the instructions_list as opposed to instructions, Could you propose the changes to this scraper so I can get a better idea of how to handle it in the future?

Reply:
Sure thing.  It's tricky at the moment, and I think it'd be worth trying to find an easier developer interface for these fields.  Personally: I'd like `instructions` to start returning a `list[str]` natively at some point in future, and then we can drop `instructions_list`.. but that could take a while.

Anyway: yep, I'll offer some edits for this branch soon (within the next day or two, I expect).

Reply:
Added a note in #617 to suggest that we make `instructions` return a list by default from `v15` onwards, and drop `instructions_list` entirely.

Reply:
@jknndy one more thought about `instructions` and lists, since I've been thinking about it quite a bit during refactoring and review:

The limitation of the current `instructions: str` type is that it's unclear whether a newline indicates a break between one instruction and the next, or whether the newline should be within the text of an instruction.

To provide an example:

```
Title: good recipe
Ingredients:
  * one foo
  * two bars
  * a qux
Instructions:

Step One
---
Chill the foo.
Once it's chilled, pour it onto the bar.

Step Two
---
Crumble the qux and combine with the foo and bar.
```

We might want to omit the titles `Step One` and `Step Two` from the output of an `instructions` call.  Maybe not always, but on some websites those kind of headings can get repetitive and spammy.

At the moment, though, it's tricky to represent this recipe using newlines in a way that would retain the (genuine) newline in the details about what to do with the `foo`.

I don't know how often that matters: those two sentences are equally meaningful/meaningless if simply placed onto a single line.  But it could provide a bit more flexibility, especially in cases where an instruction is quite lengthy in text, and has more than one paragraph.

In future by using a `list[str]` type we'd be able to represent multiline instructions.  But it's still OK if we simply split on every newline -- the results might not be as authentic but it'll still be usable, and in my opinion that's roughly what we should be aiming for (making recipe information accessible, ensuring it's attributed correctly (authorship), and then improving the accuracy and authenticity of it.  that's a lot of `a` words).